ci-lua

```diff
@@ -2,24 +2,42 @@
 
 . ./config
 
-rm -rf "$TEST_OUT_DIR/lua"
-mkdir -p "$TEST_OUT_DIR/lua"
+ALLOWED_LUA_VERSIONS='5.1 5.2 5.3'
+CURRENT_LUA_VERSION=
+
+# try to use $VARIETY from env
+for allowed_version in $ALLOWED_LUA_VERSIONS; do
+	if [ "${allowed_version}" = "${VARIETY}" ]; then
+		CURRENT_LUA_VERSION="${VARIETY}"
+	fi
+done
+
+if [ -z "${CURRENT_LUA_VERSION}" ]; then
+  # fallback to version 5.1
+  CURRENT_LUA_VERSION="5.1"
+fi
```

Comment:
I'd rather use only `lua` command in this script (~not `lua${CURRENT_LUA_VERSION}`~). The `ci-*` scripts are not exactly designed for selecting the language compiler/interpreter like you did here. This should be done in [`.travis.yml`](https://github.com/kaitai-io/ci_targets/blob/4de55e0c64ea4b1136445e21d7fd6ed053220bc9/.travis.yml#L148-L156) ideally by installing the right `apt` packages or running some commands, eventually in [`prepare-lua`](https://github.com/kaitai-io/ci_targets/blob/4de55e0c64ea4b1136445e21d7fd6ed053220bc9/prepare-lua) script.

If running this ([`prepare-lua:13`](https://github.com/kaitai-io/ci_targets/blob/4de55e0c64ea4b1136445e21d7fd6ed053220bc9/prepare-lua#L13))  will make available only the `lua5.3` command and not `lua` (I don't know, that's a question):

```bash
hererocks lua_install -r^ -l${VARIETY}
```

Can you rather add an alias or symlink the `lua5.3` executable (in `.travis.yml` build matrix or `prepare-lua`) to be available also as `lua`?

Reply:
Well, the Lua binary selection was already in place, I've only modified it to reflect on Travis env variable VARIETY
see https://github.com/kaitai-io/kaitai_struct_tests/blob/master/ci-lua#L13

If you want to change the way it was, i've just tested and hererocks always installs the lua into `lua_install/bin/lua` regardless of version in my environment. If the corect lua bin detection procedure was always pointless, we can safely get rid of that.

However, it might be useful for development if multiple lua binaries are installed, not through `hererocks`, just sayin

Reply:
> If you want to change the way it was, i've just tested and hererocks always installs the lua into `lua_install/bin/lua` regardless of version in my environment. If the corect lua bin detection procedure was always pointless, we can safely get rid of that.

Yeah, if that's the case, I'd definitely prefer it.

> However, it might be useful for development if multiple lua binaries are installed, not through `hererocks`, just sayin

Well, I think it's easier to just temporarily symlink the `lua` command than rely on some magic variable, if one needs to run multiple Lua versions locally. Are the Lua versions 5.1, 5.2, and 5.3 really so different that it makes sense to test them all?

Reply:
Ok, i'll change the script to only use `lua` binary, no detection whatsoever

They are different, especially 5.1 to 5.3 is big step, and 5.2 is the target we want because of Wireshark, so I'd really like to maintain high compatibility by testing with each version separately

Reply:
Also, comparison

**5.3**
```
./run-lua
Ran 210 tests in 0.044 seconds, 195 successes, 9 failures, 6 errors
```

**5.2**
```
./run-lua
Ran 210 tests in 0.038 seconds, 191 successes, 13 failures, 6 errors
```

**5.1**
```
./run-lua
Ran 210 tests in 0.038 seconds, 190 successes, 12 failures, 8 errors
```